common/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/common/model/translator.kt

```diff
@@ -8,6 +8,19 @@ import uk.gov.justice.digital.hmpps.prisonersearch.common.nomis.OffenceHistoryDe
 import uk.gov.justice.digital.hmpps.prisonersearch.common.nomis.OffenderBooking
 import uk.gov.justice.digital.hmpps.prisonersearch.common.nomis.OffenderIdentifier
 import uk.gov.justice.digital.hmpps.prisonersearch.common.nomis.Telephone
+import kotlin.collections.filter
```

Comment:
lots of unnecessary imports?

Reply:
Thats weird, my intellij doesnt flag them as unused. But i thought the linter objected to unused imports. Will remove